proposals/target-typed-new.md

```diff
@@ -64,7 +60,7 @@ The above rules include delegates (a reference type) and tuples (a struct type).
 (int a, int b) t = new(1, 2); // "new" is redundant
 Action a = new(() => {}); // "new" is redundant
 
-(int a, int b) t = new(); // ruled out by "use of struct default constructor"
+(int a, int b) t = new(); // OK; same as (0, 0)
```

Comment:
I thought this would still be an error: `error CS8181: 'new' cannot be used with tuple type. Use a tuple literal expression instead.`

Reply:
That is a syntactical limitation only, we still can write `new ValueTuple(..)` and that's what we emit for `new()` here.

>`(int a, int b) t = new(); // OK; same as (0, 0)
`

It would be the same as `new ValueTuple<int, int>()`

Reply:
No, this one is fine because we have a real target-type of `ValueTuple`. The one that's not allowed is deconstruction assignment.

Reply:
Right, I think `ValueTuple<int, int> t = new();` is okay, but `(int, int) t = new();` is not (one should write a tuple literal instead).

Reply:
>but (int, int) t = new(); is not (one should write a tuple literal instead).

I think we can't even distinguish between the two per "tuple compatible types" rule. 

Basically, [compiler sees both as if they are the same type](https://sharplab.io/#v2:EYLgZgpghgLgrgJwgZwLQygg5hG6CeADhACaoB2EA7gDQwgBuECAlmPgD4ACATAIwBYAFC8AzAAIA3sPGzxvHlJlyVAelXiqmci3JZxAYQDKADj49RIcQBUAFhHHxCAGwcRXAWwjkY48lC9xAHIWIPEWZHCscgB7JBJxYAgAYyg4ZAcocRI2SCQfPwCIZRVZdXE4vxjCwIjxZGJkthZSRPxHe0dMHF8YIgcggApdGBpwnwBKIIA6EtKANShnOAhrOBcIAB4RmhGAPkdxAF5hkD4xngmAbjmAX2FboA==).

(btw this was brought up in an LDM before and decided to be allowed)

Reply:
Those two examples have precisely the same target type, so both would work.

Reply:
Hum, I guess that's okay and possibly unavoidable.
I added a note to issue tracking [ExplicitType refactoring ](https://github.com/dotnet/roslyn/issues/42554) work for `new()` as it should specially handle tuples